streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -339,6 +348,11 @@
                     CommonClientConfigs.DEFAULT_SECURITY_PROTOCOL,
                     Importance.MEDIUM,
                     CommonClientConfigs.SECURITY_PROTOCOL_DOC)
+            .define(DEFAULT_DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG,
+                Type.CLASS,
+                LogAndFailExceptionHandler.class.getName(),
```

Comment:
They all have `getName`. Should I remove all? Otherwise it will look inconsistent.